tests/data/LF_ETRS89_UseCase/reference_mergeMinLoops12/inflow_daily/chanqWin.tss

```diff
@@ -0,0 +1,34 @@
+timeseries scalar settingsfile: /perm/mo/mocm/prg/oslisflood/ecmwf/lisflood/tests/data/LF_ETRS89_UseCase/settings/inflow.xml_9d48fa1c-1f8b-4b44-a769-2ff550e727d9.xml date: Wed Feb 17 16:11:05 2021
```

Comment:
We can delete the folder "tests/data/LF_ETRS89_UseCase/reference_mergeMinLoops12" before the push